docs/concepts/llm_convenience_wrappers.md

```diff
@@ -18,14 +18,14 @@ You can initialize an [`OpenAIChat`](../api/chat/models.md#mirascope.chat.models
 ```python
 from mirascope import OpenAIChat, Prompt
 
-class RecipePrompt:
-		"""
-		Recommend recipes that use {ingredient} as an ingredient
-		"""
+class RecipePrompt(Prompt):
+	"""
+	Recommend recipes that use {ingredient} as an ingredient
+	"""
 
-		ingredient: str
+	ingredient: str
 
-chat = OpenAIChat()
+chat = OpenAIChat(api_key="YOUR_OPENAI_API_KEY")
```

Comment:
here and elsewhere, do we want to do this or the os environ setting like in our examples? I'm leaning toward the latter, wdyt?

Reply:
i think for concepts its easier to understand that this is just an openai wrapper. for cookbook, thats what we intend for production use, e.g. use os and environment variables